src/core/cita_bft.rs

```diff
@@ -146,6 +146,7 @@ pub struct Bft {
     block_txs: VecDeque<(usize, BlockTxs)>,
     block_proof: Option<(usize, BlockWithProof)>,
 
+    // The verified blocks with tx body.
     verified_blocks: HashMap<H256, Block>,
```

Comment:
So this only can be inserted when receiving the Auth's verified result?

Reply:
Yes, I have found it after reviewed all. But I think that you should write more comments at here, as a newbie it's too hard to infer all the things.

Reply:
Writing one thing in two locations is redundant, and redundancies always bring ambiguity.
For high maintainability, comments should be always in the locations where the associated codes are.

For example, publishers send messages and subscribers subscribe message.
I was puzzled why there are lists of subscribers in the locations before definitions of publishers.
Do publishers CARE ABOUT who subscribe them?
So how can we decouple them?

I do NOT think more comments always are right things.

In other scenes like this but more complex, I think the best way is to provide different methods to do those.